packages/browser-crawler/src/internals/browser-crawler.ts

```diff
@@ -217,13 +222,14 @@ export interface BrowserCrawlerOptions<
     /**
      * Async functions that are sequentially evaluated after the navigation. Good for checking if the navigation was successful.
      * The function accepts `crawlingContext` as the only parameter.
-     * Example:
-     * ```
+     *
+     * @example
```

Comment:
not sure if we want to use the `@example` tag. docusaurus renders it [this way](https://crawlee.dev/api/basic-crawler/interface/CrawlerAddRequestsResult#waitForAllRequestsToBeAdded), might be better to keep it like before

also we use it exactly ones in the whole code base

Reply:
Yeah, I saw it used in some parts, thougth that we decided to go this way at some point =D I do not like it either, so would revert. I do not like the `@default` either, but default values are usually something that does not take half of the screen + it makes it kinda more visible, so would probably leave defaults this way

Reply:
I hope the `@default` tag will get some love in the docusaurus plugin and will get rendered better in future versions.